src/api/app/controllers/webui/search_controller.rb

```diff
@@ -2,6 +2,7 @@ class Webui::SearchController < Webui::WebuiController
   before_action :set_attribute_list
   before_action :set_tracker_list
   before_action :set_parameters, except: :issue
+  skip_before_action :verify_authenticity_token, only: [:index]
```

Comment:
Why is this a `POST` route at all? We are not changing/storing any data?

Reply:
:facepalm: true

Reply:
its allows both POST and GET, I will check

Reply:
@hennevogel I changed all the POST' to GET', no idea why we used POST', couldn't find a reason for it